### PR TITLE
[MKT-461]: Updating cloud-object-storage

### PR DESCRIPTION
**Pull request by jaaaaavier**
1. Removed useDebounce implement and internxt logo from graphcomponent
2. Created JSONs for the rest of the languages

### CODE REVIEW

File: src/pages/cloud-object-storage/index.tsx [hunk 1]
```diff
@@ -83,7 +92,7 @@ export function getServerSideProps(ctx: GetServerSidePropsContext) {
```
Comment (xabg2) [inline src/pages/cloud-object-storage/index.tsx:92]: Use locale for all JSON as we have the translations for the page.

File: src/components/cloud-object-storage/HowMuchYouNeedSection.tsx [hunk 2]
```diff
@@ -55,8 +55,6 @@ export const HowMuchYouNeedSection = ({ textContent }: HowMuchYouNeedSectionProp
   const [storageAmountValue, setStorageAmountValue] = useState<number>(500);
   const [percentDownloadValue, setPercentDownloadValue] = useState<number>(2);
 
-  const debouncedStorageAmountValue = useDebounce(storageAmountValue, 500);
```
Comment (xabg2) [inline src/components/cloud-object-storage/HowMuchYouNeedSection.tsx:58]: Remove the `useDebounce` import

**Review by xabg2 — CHANGES_REQUESTED** [review]

File: src/pages/cloud-object-storage/index.tsx (hunk 1 above)
Comment (jaaaaavier) [reply src/pages/cloud-object-storage/index.tsx:92]: Done

File: src/components/cloud-object-storage/HowMuchYouNeedSection.tsx (hunk 2 above)
Comment (jaaaaavier) [reply src/components/cloud-object-storage/HowMuchYouNeedSection.tsx:58]: Done

**Review by xabg2 — APPROVED** [review]

**Pull request merged**